style.css

```diff
@@ -0,0 +1,60 @@
+body {
+    width: 500px;
+    height: 100%;
+    margin: 5px auto;
+    background-color: lightgray;
+}
+.container {
```

Comment:
- [ ]  Kindly make sure to add a new line after the` }` to look your css code professional